pandas/core/window.py

```diff
@@ -2775,6 +2788,8 @@ def _get_center_of_mass(comass, span, halflife, alpha):
 
 
 def _offset(window, center):
+    # TODO: (MATT) If the window is a BaseIndexer subclass,
+    #  we need to pass in the materialized window
```

Comment:
What is the type of window here? It looks like it can be a sequence or an integer?

Reply:
Correct. After a light audit, I anticipate the materialized window may be passed here and other times and `Indexer` class might be passed here.

Overall I think this routine is for label formatting.